### ISSUE TITLE
Sniffle

### ISSUE THREAD

**Issue by noraj** [BlackArch/blackarch#2830]
- **Name**: Sniffle
- **Source URL**: https://github.com/nccgroup/sniffle

**Comment by noptrix** [comment]
@noraj any news on this? if not let's close it please

**Comment by noraj** [comment]
no a tool request I opened on behalf of someone else

**Comment by D3vil0p3r** [comment]
https://github.com/BlackArch/blackarch-pkgbuilds/issues/21

Btw it should be useful, for tracking purpose, to create a new label `container` for those tool request issues related to tools deployed as containers.

**Comment by noraj** [comment]
> [BlackArch/blackarch-pkgbuilds#21](https://github.com/BlackArch/blackarch-pkgbuilds/issues/21)
> 
> Btw it should be useful, for tracking purpose, to create a new label `container` for those tool request issues related to tools deployed as containers.

Seem totally unrelated to this issue?

**Comment by D3vil0p3r** [comment]
Yes, because I smoked something weird that day. My comment was related to another issue...

**Comment by D3vil0p3r** [comment]
https://github.com/BlackArch/blackarch/pull/4349 I need just some suggestions...

This package uses `arm-none-eabi-gcc` so it can be built only on ARM cpu-based systems. Its dependency [uniflash](https://aur.archlinux.org/packages/uniflash) seems to be only for `i686` and `x86_64` by looking its AUR PKGBUILD... So how can an ARM-based tool working with a non-ARM dependency?

Please, if you have an ARM-based device, can you test if the build goes well?
